application/src/main/java/org/opentripplanner/model/plan/walkstep/verticaltransportationuse/VerticalTransportationUse.java

```diff
@@ -0,0 +1,7 @@
+package org.opentripplanner.model.plan.walkstep.verticaltransportationuse;
+
+/**
+ * Represents information about vertical transportation equipment stored in
+ * {@WalkStep}.
+ */
+public interface VerticalTransportationUse {}
```

Comment:
We probably need @t2gran's opinion on the use of an empty interface for these. My guess is that he prefers if they were just separate fields instead of using one field that uses the interface.